src/desktop/bugsnag.js

```diff
@@ -16,4 +17,16 @@ reportBuild(
     },
 )
     .then(() => console.log('Build successfully reported to Bugsnag'))
-    .catch((err) => console.log('Build could not be reported to Bugsnag: ', err.messsage));
+    .catch((err) => console.log('Build could not be reported to Bugsnag: ', err));
+
+upload({
+    apiKey: API_KEY,
+    appVersion: version,
+    publicPath: 'iota://dist/',
+    minifiedUrl: 'iota://dist/bundle.js',
+    sourceMap: resolve(__dirname, 'dist/bundle.js.map'),
+    minifiedFile: resolve(__dirname, 'dist/bundle.js'),
+    overwrite: true,
+})
+    .then(() => console.log('Sourcemaps successfully uploaded to Bugsnag'))
+    .catch((err) => console.log('Sourcemaps could not uploaded to Bugsnag: ', err));
```

Comment:
Just a typo:
could not ** be ** uploaded

Reply:
Good catch, thanks @dangenendt !